src/store/tasks/tasks-context.js

```diff
@@ -0,0 +1,36 @@
+import { createContext, useState, useContext } from 'react';
+
+const TaskContext = createContext();
+
+export const TaskContextProvider = ({ children }) => {
+  const [showMemberTaskUpdateModal, setShowMemberTaskUpdateModal] =
+    useState(false);
+  const [isNoteworthy, setIsNoteworthy] = useState(false);
+  const [isOptionKey, setIsOptionKey] = useState(false);
```

Comment:
This will be removed after this issue is closed
- https://github.com/Real-Dev-Squad/website-members/issues/420